app/models/web_hook_event_type.rb

```diff
@@ -19,6 +19,15 @@ class WebHookEventType < ActiveRecord::Base
   default_scope { order('id ASC') }
 
   validates :name, presence: true, uniqueness: true
+
+  def self.all_excluding_disabled_plugin_web_hooks
+    ids_to_exclude = []
+    ids_to_exclude << SOLVED unless defined?(SiteSetting.solved_enabled) && SiteSetting.solved_enabled
```

Comment:
Hmmm 🤔 Shouldn't this be the responsability of the plugin to "add" their webhook types?

Reply:
Yes, originally that was the plan

https://github.com/discourse/discourse-assign/pull/61#discussion_r385752247

But if we reserve the webhook id here we prevent future id conflicts.